### PR TITLE
Add new commercial tools

### PR DESCRIPTION
**Pull request by zhimin-z**
add a few commercial tools

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -552,11 +555,13 @@ This repository contains a curated list of awesome open source libraries that wi
...
 * [Iguazio Data Science Platform](https://www.iguazio.com/) - Bring your Data Science to life by automating MLOps with end-to-end machine learning pipelines, transforming AI projects into real-world business outcomes, and supporting real-time performance at enterprise scale.
 * [Iterative Studio](https://studio.iterative.ai/) - Seamless data and model management, experiment tracking, visualization and automation, with Git as the single source of truth.
 * [Katonic.ai](https://katonic.ai/) - Automate your cycle of Intelligence with Katonic MLOps Platform.
 * [Labelbox](https://labelbox.com/) - Image labelling service with support for semantic segmentation (brush & superpixels), bounding boxes and nested classifications.
 * [Microsoft Azure Machine Learning service](https://azure.microsoft.com/en-us/services/machine-learning-service/) - Build, train, and deploy models from the cloud to the edge.
+Our breakthrough technology combines the skills and expertise of the human user, cutting edge neuro-technology and state of the art machine learning and signal processing algorithms.
```
Comment (axsaucedo) [inline README.md:564]: Not sure what this is

File: README.md [hunk 2]
```diff
@@ -529,11 +529,14 @@ This repository contains a curated list of awesome open source libraries that wi
 
 
 ## Commercial Platform
+* [Amazon Augmented AI](https://aws.amazon.com/augmented-ai/) - Amazon Augmented AI (A2I) allows you to conduct a human review of machine learning (ML) systems to guarantee precision.
```
Comment (axsaucedo) [inline README.md:532]: I'd be keen to merge all AWS to one, ideally single product per company in this section, we may need to update the contribting file

**Review by axsaucedo — CHANGES_REQUESTED** [review]
ADded changes requested

File: README.md (hunk 2 above)
Comment (zhimin-z) [reply README.md:532]: sorry that is typo

Comment (zhimin-z) [reply README.md:532]: I wonder how we can unite all AI-related products into one, may use Amazon directly? @axsaucedo

**Comment by zhimin-z** [comment]
I wonder how we can unite all AI-related products into one, may use Amazon directly? @axsaucedo

**Comment by axsaucedo** [comment]
Yes I agree it's a bit of a hard one, perahps we can make an entry like:
```
* AWS AI Ecosystem - The Amazon Web Services provides a broad set of AI services, including [Sagemaker](link), [Foo](link], [Bar](link], etc...
```

That shoudl be enough to encompass it - if other companies have multipel ones we can do the same or similar

**Comment by zhimin-z** [comment]
> Yes I agree it's a bit of a hard one, perahps we can make an entry like:
> 
> ```
> * AWS AI Services / Ecosystem - The Amazon Web Services provides a broad set of AI services, including [Sagemaker](link), [Foo](link], [Bar](link], etc...
> ```
> 
> That shoudl be enough to encompass it - if other companies have multipel ones we can do the same or similar

@axsaucedo Solved!

**Comment by axsaucedo** [comment]
Thank you @zhimin-z but it still seems my concern above of haivng large corps occupying too much space on the list, i would rather limit one entry per company and group them as suggested (ie only name of tool+link)

**Comment by zhimin-z** [comment]
> Thank you @zhimin-z but it still seems my concern above of haivng large corps occupying too much space on the list, i would rather limit one entry per company and group them as suggested (ie only name of tool+link)

Hmm.... I have a question. Since different tools aim for different purposes, why not include them with a short description for each? What do you think? @axsaucedo

**Comment by axsaucedo** [comment]
My concern is that there would just betoo much space for large corporations, whereas smaller companies like startups that offer a single service would only have one block. I don't think at this stage people are struggling to learn about AWS AI services (as they have a huge marketing team behind). I would be keen to make sure this is the case with other cloud providers like Google, Azure, etc.

That's the thinking behind trying to avoid being very descriptive as well, as also if people are interested they can have a look - the commercial list has always been an outlier as its' not clear if the products are still going, widely used, etc (as there's no stars / active maintenance equivallent, etc)

**Comment by zhimin-z** [comment]
> My concern is that there would just betoo much space for large corporations, whereas smaller companies like startups that offer a single service would only have one block. I don't think at this stage people are struggling to learn about AWS AI services (as they have a huge marketing team behind). I would be keen to make sure this is the case with other cloud providers like Google, Azure, etc.
> 
> That's the thinking behind trying to avoid being very descriptive as well, as also if people are interested they can have a look - the commercial list has always been an outlier as its' not clear if the products are still going, widely used, etc (as there's no stars / active maintenance equivallent, etc)

Make sense!

File: README.md [hunk 3]
```diff
@@ -529,11 +529,14 @@ This repository contains a curated list of awesome open source libraries that wi
 
 
 ## Commercial Platform
-* [Amazon SageMaker](https://aws.amazon.com/sagemaker/) - End-to-end machine learning development and deployment interface where you are able to build notebooks that use EC2 instances as backend, and then can host models exposed on an API.
+* [Amazon Web Services](https://aws.amazon.com/) - Amazon Web Services offers reliable, scalable, and inexpensive cloud computing services to productionalize machine learning, including: [Amazon Augmented AI](https://aws.amazon.com/augmented-ai/), [Amazon Rekognition](https://aws.amazon.com/rekognition/), [Amazon SageMaker](https://aws.amazon.com/sagemaker/).
```
Comment (axsaucedo) [inline README.md:532]: This looks better, I would remove inexpensive as that's not something I've heard describe organisations running SageMaker at scale (if anything, the opposite, but other than that looks good

File: README.md [hunk 4]
```diff
@@ -548,21 +551,27 @@ This repository contains a curated list of awesome open source libraries that wi
...
+* [Fiddler](https://www.fiddler.ai/) - Fiddler is a model performance management platform that offers model monitoring, observability, explainability & fairness.
 * [Google Cloud Machine Learning Engine](https://cloud.google.com/ml-engine/) - Managed service that enables developers and data scientists to build and bring machine learning models to production.
 * [Graphsignal](https://graphsignal.com/) - Machine learning profiler that helps make model training and inference faster and more efficient.
 * [H2O Driverless AI](https://www.h2o.ai/products/h2o-driverless-ai/) - Automates key machine learning tasks, delivering automatic feature engineering, model validation, model tuning, model selection and deployment, machine learning interpretability, bring your own recipe, time-series and automatic pipeline generation for model scoring - [(Video)](https://www.youtube.com/watch?v=ZqCoFp3-rGc).
+* [Hugging Face](https://huggingface.co/) - Hugging Face is a platform that allows users to share machine learning models and datasets.
```
Comment (axsaucedo) [inline README.md:558]: HuggingFace is one word

File: README.md [hunk 5]
```diff
@@ -571,6 +580,10 @@ This repository contains a curated list of awesome open source libraries that wi
...
 * [Talend Studio](https://www.talend.com/) - Data integration platform that provides various software and services for data integration, data management, enterprise application integration, data quality, cloud storage and Big Data.
+* [Tecton](https://www.tecton.ai/) - Tecton is an all-in-one system to build, automate, and centralize feature workflows for production ML.
 * [Valohai](https://valohai.com/) - Machine orchestration, version control and pipeline management for deep learning.
 * [Vertex AI](https://cloud.google.com/vertex-ai) - Vertex AI Workbench is the single environment for data scientists to complete all of their ML work, from experimentation, to deployment, to managing and monitoring models. It is a Jupyter-based fully managed, scalable, enterprise-ready compute infrastructure with security controls and user management capabilities.
+* [Ultralytics](https://ultralytics.com/) Ultralytics simplifies data sourcing, labeling, model training, and deployment services that offer artificial intelligence.
```
Comment (axsaucedo) [inline README.md:586]: Seems only has 9 employees on linkedin, great to support startups by adding to the list but keen ot keep an eye on these as we've had cases in the past where these disappear or change their domain very drastically

**Review by axsaucedo — CHANGES_REQUESTED** [review]
Thank you for the effort @zhimin-z , this seems like a great addition, I added a few minor comments and then good to merge

File: README.md (hunk 4 above)
Comment (zhimin-z) [reply README.md:558]: I initially thought so, but it is actually two words. @axsaucedo 
image

File: README.md (hunk 5 above)
Comment (zhimin-z) [reply README.md:586]: Sure, but this is a popular startup recently along with generative AI.

File: README.md (hunk 3 above)
Comment (zhimin-z) [reply README.md]: Done

File: README.md (hunk 5 above)
Comment (axsaucedo) [reply README.md:586]: There's going to be a lot of popular GenAI startups in the next couple of months, this goes back to the https://github.com/EthicalML/awesome-production-machine-learning/blob/master/CONTRIBUTING.md guidelines that we're not trying to make this an `This is not an exhaustive listing` but a curated listing. I think adding to the CONTRIBUTING.md that startups should have some thresholds similar to OSS projects (ie min 15 employees, and 1 year going)

Comment (zhimin-z) [reply README.md:586]: Ok ok, but if you take a look at their OSS project profile: their YOLOv[4-8] has a very high impact in the CV domain and the contributors are a lot (~50), the star number seems good too (~4k). @axsaucedo I think Ultralytics is indeed a company to be written a bit here.
image

Comment (axsaucedo) [reply README.md:586]: Ok that's actually great to hear - similar to the contributing.md there will certainly be exceptions, and this seems like one of them, appreciate providing these overviews. LGTM

**Pull request merged**